### DISCUSSION TITLE
I am looking for certain key bindings

### DISCUSSION THREAD

**Discussion by vgnshiyer** [zed-industries/zed#8292] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Love the new editor. I was looking for below key bindings from vscode. It would be great to have them.

1.  workbench.action.moveEditorToNextGroup --> Moves current focussed window to a split screen on the right.
2. workbench.action.terminal.split --> Splits the terminal by opening a new one to the right.
3. workbench.action.terminal.resizePaneUp --> resizes the terminal upwards.
4. Copilot chat feature from vscode. 

_No response_

**Comment by Moshyfawn** [comment]
| VSC                                      | Zed                                                   |
|------------------------------------------|-------------------------------------------------------|
| `workbench.action.moveEditorToNextGroup` | `cmd-k <direction>` _(`up`, `down`, `left`, `right`)_ |
| `workbench.action.terminal.split`        | N/A _(can only add another terminal)_                 |
| `workbench.action.terminal.resizePaneUp` | `shift-escape`                                        |

The Copilot Chat feature request is being tracked in #4673. At the moment there is only the assistant panel with multiple ChatGPT model integration.
